ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -182,6 +182,7 @@ func (r *RayClusterReconciler) rayClusterReconcile(ctx context.Context, request
 	// variable `ENABLE_GCS_FT_REDIS_CLEANUP` to `false`, and undertake the Redis storage namespace cleanup
 	// manually after the RayCluster CR deletion.
 	enableGCSFTRedisCleanup := strings.ToLower(os.Getenv(utils.ENABLE_GCS_FT_REDIS_CLEANUP)) != "false"
+	deadlineGCSFTRedisCleanup := utils.EnvInt64(utils.GCS_FT_REDIS_CLEANUP_DEADLINE_SECONDS, 300)
```

Comment:
This variable seems only to be used by `buildRedisCleanupJob`. We can read the env var in `buildRedisCleanupJob`.